apis/r/R/TileDBGroup.R

```diff
@@ -415,7 +451,8 @@ TileDBGroup <- R6::R6Class(
       }
 
       if (private$.mode == "WRITE") {
-        tiledb::tiledb_group_close(group_handle)
+        #tiledb::tiledb_group_close(group_handle)
+        c_group_close(group_handle)
```

Comment:
Can we clean out the commented code?
```suggestion
        c_group_close(group_handle)
```